ush/config_defaults.sh

```diff
@@ -295,6 +296,7 @@ OBSPATH_NSSLMOSIAC="/public/data/radar/mrms"
 LIGHTNING_ROOT="/public/data/lightning"
 ENKF_FCST="/lfs4/BMC/public/data/grids/enkf/atm"
 FFG_DIR="/public/data/grids/ncep/ffg/grib2"
+ARI_DIR="/lfs4/BMC/wrfruc/ejames/datasets/ari/erickson"
```

Comment:
@EricJames-NOAA  How about setting FFG_DIR ="" and ARI_DIR="" by default so that this can be used to skip the corresponding logic in scripts/exregional_run_post.sh? And then if one wants to have FFG and ARI outputs, FFG_DIR and ARI_DIR has to be explicitly specified in config.sh.

Reply:
@guoqing-noaa thanks for the idea.  So you propose adding an extra check on FFG_DIR and ARI_DIR existing?  I can make that modification

Reply:
Yes, that will be good for RTMA or other applications. Thanks!